configs/perlmutter.json

```diff
@@ -1,21 +1,21 @@
 {
-    "cms_dir": "/global/homes/m/moraru/Parsl-Project/exa-amd/cms_dir",
-    "work_dir": "/pscratch/sd/m/moraru/work_dir_v3",
-    "cpu_account": "m4802",
-    "gpu_account": "m4802_g",
+    "cms_dir": "<path_to>/exa-amd/cms_dir",
+    "work_dir": "./work_dir_v1",
+    "cpu_account": "cpu_account",
+    "gpu_account": "gpu_account",
     "elements": "Na-B-C",
     "formation_energy_threshold": -0.2,
     "num_workers": 128,
-    "initial_structures_dir":"/global/homes/m/moraru/Parsl-Project/exa-amd/cms_dir/mpstrs",
+    "initial_structures_dir":"initial_structures",
```

Comment:
Same as above. Is it only the name of the directory or is it a path? If it is a path, change it to "./initial_structures" or "<path_to>/initial_structures".